glide.yaml

```diff
@@ -7,10 +7,11 @@ import:
   - arm/network
   - arm/resources/resources
 - package: github.com/Azure/go-autorest
-  version: ~8.0.0
+  version: master
```

Comment:
Is this because there's no tagged release with the latest changes?  If so I think we should create a tag and lock it down to that instead.